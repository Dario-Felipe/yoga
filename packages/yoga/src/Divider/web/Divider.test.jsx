import React from 'react';
import { render } from '@testing-library/react';

import { ThemeProvider, Divider, Box, Text } from '../..';

describe('<Divider />', () => {
  it('should create a divider component', () => {
    const { container } = render(
      <ThemeProvider>
        <Box flex={1} display="flex" flexDirection="column" minHeight={50}>
          <Text.Regular>First Content</Text.Regular>
          <Divider />
          <Text.Regular>Second Content</Text.Regular>
        </Box>
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });

  it('should create a vertical divider component', () => {
    const { container } = render(
      <ThemeProvider>
        <Box flex={1} display="flex" flexDirection="row" minHeight={50}>
          <Text.Regular>First Content</Text.Regular>
          <Divider vertical />
          <Text.Regular>Second Content</Text.Regular>
        </Box>
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
