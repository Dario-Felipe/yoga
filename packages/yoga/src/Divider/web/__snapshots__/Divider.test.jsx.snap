// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should match snapshot 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  height: 1px;
  background-color: #D7D7E0;
}

<div>
  <div
    class="c0"
  />
</div>
`;
